opengrok-indexer/src/main/java/org/opengrok/indexer/web/PageConfig.java

```diff
@@ -794,24 +794,28 @@ public String getRequestedProjectsAsString() {
      * <p>
      * NOTE: This method assumes, that project names do <b>not</b> contain a
      * comma (','), since this character is used as name separator!
+     * <p>
+     * It is determined as follows:
+     * <ol>
+     * <li>If there is no project in the configuration an empty set is returned. Otherwise:</li>
+     * <li>If there is only one project in the configuration,
+     * this one gets returned (no matter, what the request actually says). Otherwise</li>
+     * <li>If the request parameter {@code project} contains any available project,
+     * the set with invalid projects removed gets returned. Otherwise:</li>
+     * <li>If the request has a cookie with the name {@code OpenGrokProject}
+     * and it contains any available project,
+     * the set with invalid projects removed gets returned. Otherwise:</li>
+     * <li>If a default project is set in the configuration,
+     * this project gets returned. Otherwise:</li>
+     * <li>an empty set</li>
+     * </ol>
      *
-     * @return a possible empty set of project names but never
-     * {@code null}. It is determined as follows: <ol> <li>If there is no
-     * project in the runtime environment (RTE) an empty set is returned.
-     * Otherwise:</li> <li>If there is only one project in the RTE, this one
-     * gets returned (no matter, what the request actually says). Otherwise</li>
-     * <li>If the request parameter {@code project} contains any available
-     * project, the set with invalid projects removed gets returned.
-     * Otherwise:</li> <li>If the request has a cookie with the name
-     * {@code OpenGrokProject} and it contains any available project, the set
-     * with invalid projects removed gets returned. Otherwise:</li> <li>If a
-     * default project is set in the RTE, this project gets returned.
-     * Otherwise:</li> <li>an empty set</li> </ol>
+     * @return a possible empty set of project names but never {@code null}.
      */
     public SortedSet<String> getRequestedProjects() {
         if (requestedProjects == null) {
             requestedProjects
-                    = getRequestedProjects("project", OPEN_GROK_PROJECT);
+                    = getRequestedProjects("project", "group", OPEN_GROK_PROJECT);
```

Comment:
define "group" as this final static string ?

Reply:
the same for "project" ? I kept it consisten but I can improve it

Reply:
yep

Reply:
I did the same for project